src/main.rs

```diff
@@ -263,6 +265,10 @@ enum Commands {
         #[arg(short = 'm', long)]
         match_seq_num: Option<u64>,
 
+        /// Input format. Can be one of "text" or "json".
```

Comment:
Does clap already show valid options? Missing `json-binsafe` here

Reply:
Clap doesn't show valid options. Will take a look if possible, else will add in the comment.

Reply:
Yup, that's possible. Made the change.